client/src/components/EditAccount.css

```diff
@@ -0,0 +1,9 @@
+Label {
+    width: 125px;
+    text-align: right;
+}
+
+Button {
+    margin: 8px;
+    width: 200px;
+}
```

Comment:
Are These CSS changes for EVERY Label and Button on the entire website? Otherwise, we should use classes. I will discuss this and naming convention in class tomorrow. If you'd like to merge it in and refactor later, that's fine, but we shouldn't forget to.

Reply:
They are just for the EditAccount component. Does that still need to be refactored/use classes? Or should I merge this into master?